eng/pipelines/templates/steps/build-conda-artifacts.yml

```diff
@@ -0,0 +1,77 @@
+parameters:
+  - name: TestPipeline
+    type: boolean
+    default: false
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: CondaArtifacts
+    type: object
+    default: []
+
+steps:
+  - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+    parameters:
+      PackageName: "azure-template"
+      ServiceDirectory: "template"
+      TestPipeline: ${{ parameters.TestPipeline }}
+
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(PythonVersion)'
+    inputs:
+      versionSpec: $(PythonVersion)
+
+  - script: |
+      pip install -r eng/ci_tools.txt
+    displayName: 'Prep Environment'
+
+  - pwsh: |
+      mkdir $(Agent.BuildDirectory)/conda/
+      mkdir $(Agent.BuildDirectory)/conda/output
+      mkdir $(Agent.BuildDirectory)/conda/build
+
+      Write-Host "##vso[task.setvariable variable=conda.output]$(Agent.BuildDirectory)/conda/output"
+      Write-Host "##vso[task.setvariable variable=conda.build]$(Agent.BuildDirectory)/conda/build"
+    displayName: 'Create Conda Working Directories'
+
+  - ${{ each artifact in parameters.CondaArtifacts }}:
+    # there may be multiple CondaArtifacts. Be certain $(conda.build) is clean just in case!
+    - pwsh:
+        Write-Host "Clean up Conda Build Directory $(conda.build)"
+        Remove-Item $(conda.build)/* -Recurse -Force
+      displayName: 'Clean Up Before Building ${{ artifact.name }}'
+
+    - ${{ each checkout in artifact.checkout }}:
+      - template: /eng/pipelines/templates/steps/get-tagged-code.yml
```

Comment:
Maybe I miss-understood yesterday but didn't you say you plan to stop getting and building code but instead just downloading the artifacts from some cache?

Reply:
This is to assemble the combined `sdist`. The cache I was talking about is for the prebuilt `conda` dependencies.

https://azuresdkconda.blob.core.windows.net/channel1/index.html